packages/react-notion-x/src/components/asset.tsx

```diff
@@ -217,13 +212,20 @@ export const Asset: React.FC<{
     const caption = getTextContent(block.properties?.caption)
     const alt = caption || 'notion image'
 
+    // Generate RGB-based base64 images for image placeholder while loading
+    const keyStr = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/='
```

Comment:
L215-L218 don't belong in a production library.

The current way the preview image is handled is by using `recordMap.preview_images` to store a map of image source URLs to preview image URLs.

Reply:
Just made a new commit and removed the preview placeholder! d235b40